<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>JobManagerDialog</class>
 <widget class="QDialog" name="JobManagerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NanoEngineer-1 Job Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>Jobs</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QTableWidget" name="job_table" >
        <property name="numRows" >
         <number>1</number>
        </property>
        <property name="numCols" >
         <number>8</number>
        </property>
        <property name="sorting" >
         <bool>false</bool>
        </property>
        <property name="selectionMode" >
         <enum>QTable::SingleRow</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="start_btn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_btn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_btn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="view_btn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_btn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="move_btn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Move</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>280</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="refresh_btn" >
       <property name="text" >
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filter_btn" >
       <property name="text" >
        <string>Filter...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>JobManagerDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
